serialization/README.md

```diff
@@ -2,13 +2,36 @@
 
 ## Serialization info
 
-_TBD_
+This directory contains the specifications and examples for all supported SPDX formats.
+
+There is a specification markdown file for each supported format describing the serialization.
+
+A collection of elements can be serialized in multiple formats.
+
+Within the model, we have an "XCollection" which represents the common properties of the collection of elements across all data formats.
+
+We can refer to the actual serialized bytes as and Artifact in the model.
```

Comment:
Suggest slight tweak to "We can represent/characterize the actual serialized bytes using an Artifact element in the model."